documentation/1.11/content/userguide/tools/update-image.md

```diff
@@ -23,47 +23,48 @@ Usage: imagetool update [OPTIONS]
+| `--resourceTemplates` | One or more files containing placeholders that need to be resolved by the Image Tool. See [Resource Template Files](#resource-template-files).                                                                                      |                                |
+| `--skipcleanup` | Do not delete the build context folder, intermediate images, and failed build containers. For debugging purposes.                                                                                                                   |                                |
+| `--strictPatchOrdering` | Instruct OPatch to apply patches one at a time (uses `apply` instead of `napply`).                                                                                                                                                  |                                |
+| `--target` | Select the target environment in which the created image will be used. Supported values: `Default` (Docker/Kubernetes), `OpenShift`. See [Additional information](#--target).                                                       | `Default`                      |
+| `--type` | Installer type. Supported values: `WLS`, `WLSDEV`, `WLSSLIM`, `FMW`, `IDM`, `OSB`, `OUD_WLS`, `SOA_OSB`, `SOA_OSB_B2B`, `MFT`, `WCP`, `OAM`, `OIG`, `OUD`, `OID`, `SOA`, `WCC`, `WCS`, `WCP`                                        | Installer used in `fromImage`  |
+| `--user` | Oracle support email ID. When supplying `user`, you must supply the password either as an environment variable using `--passwordEnv`, or as a file using `--passwordFile`, or interactively, on the command line with `--password`.   |                                |
+| `--wdtArchive` | A WDT archive ZIP file or comma-separated list of files.                                                                                                                                                                            |                                |
+| `--wdtDomainHome` | Path to the `-domain_home` for WDT.                                                                                                                                                                                                 | `/u01/domains/base_domain`     |
+| `--wdtDomainType` | WDT domain type. Supported values: `WLS`, `JRF`, `RestrictedJRF`                                                                                                                                                                    | `WLS`                          |
+| `--wdtEncryptionKey` | Passphrase for WDT -use_encryption that will be requested on STDIN.                                                                                                                                                                 |                                |
```

Comment:
-use_encryption -> `-use_encryption` (should be in code font in each of the three instances where it is used)